cirq/iqm/iqm_device.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IQM builds quantum computers based on superconducting transmon technology.
+For more information, see https://meetiqm.com.
+"""
+
+from typing import cast, Iterable, Set
+import cirq
+from cirq import devices, ops, protocols
+
+
+class Adonis(devices.Device):
+    """IQM's five-qubit superconducting device with pairwise connectivity.
+    The qubits are connected thus (recommend using fixed-width font for
+    viewing):
+         q0
+         |
+    q1 - q2 - q3
+         |
+         q4
+    where the lines denote which qubits can be subject to the two-qubit gate.
+
+    Each qubit can be rotated about any axis by an arbitrary angle. I.e. the
```

Comment:
I don't think these two things are the same thing.  A Hadamard (cirq.H) is rotating around an axis (X+Z) but is not X/Y/ZPowGate.  Is it supported?

Reply:
Okay, fair point, I was thinking about axes being the X, Y, and Z axes 🙂 I will rephrase.

Reply:
Rephrased.